src/test/deployments.test.js

```diff
@@ -133,6 +134,35 @@ describe("Deployments", function () {
 
 });
 
+describe("Deployments navigation badge", function () {
+
+  beforeEach(function () {
+    var props = {
+      activeTabId: "deployments",
+      activeDeployments: 3,
```

Comment:
This should come from the store.

You can override the deployments object in the store, like in the info tests.